packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -318,7 +318,27 @@ export class DocumentDeltaConnection
 	/**
 	 * Disconnect from the websocket and close the websocket too.
 	 */
-	protected closeSocket(error: IAnyDriverError) {
+	private closeSocket(error: IAnyDriverError) {
+		if (this._disposed) {
+			// This would be rare situation due to complexity around socket emitting events.
+			this.logger.sendTelemetryEvent(
```

Comment:
Should this be an error? I'm curious to get stack and understand if there are some issues we need to fix directly - error would probably have stack most of the time, but if there is no error, we will not have stack recorded unless it's an error event.

Reply:
Error is not an optional argument here. So imo, this is fine as is.

Reply:
I meant - should we use sendErrorEvent here?